expression/aggregation/window_func.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2018 PingCAP, Inc.
```

Comment:
Change the package name to `udf` or something else?

Reply:
But its behavior is quite different with normal aggregate function since it won't make the rows grouped into one though there's `OVER` clause.
I think they should be in the same level not superior-subordinate.